package.json

```diff
@@ -193,6 +193,21 @@
 					"default": true,
 					"description": "When true, the option to delete the remote will be selected by default when deleting a branch from a pull request."
 				},
+				"githubPullRequests.terminalLinks.default": {
+					"type": "string",
+					"enum": [
+						"github",
+						"vscode",
+						"ask"
+					],
+					"enumDescriptions": [
+						"Create the PR in github",
```

Comment:
To be consistent with the rest of the UI and other settings, I think these descriptions should be:
```
Create the pull request on GitHub
Create the pull request in VS Code
Ask which of the above methods to use
```

Reply:
Sure thing, I'll update them.